src/tools/bin/egetopt

```diff
@@ -149,4 +163,4 @@ while true ; do
     shift
 done    
 
-getopt $getOptArgs -- $transformedArgs
+echo "getopt $getOptArgs -- $transformedArgs" | bash
```

Comment:
I was more wondering why we don't use the absolute path in this line, too ;)

what if `/bin` is not in the `$PATH` ? ;) never mind.